packages/server/server/common/server.ts

```diff
@@ -20,7 +20,7 @@ const exit = process.exit;
 export default class ExpressServer {
     private routes: (app: Application) => void;
     constructor() {
-        const root = path.normalize(__dirname + '/../..');
+        const root = path.normalize(__dirname + '/../../..');
```

Comment:
It is a bit confusing how this points to a different directory after we build/compile, maybe we can document this haha

Reply:
Sure!